.github/workflows/makefile.yml

```diff
@@ -36,5 +36,44 @@ jobs:
     - name: Unit test
       run: cd .. && python3 tests/tools/ffmpeg.py tests/conformance/passed
 
-    - name: Negtive test
+    - name: Negative test
       run: cd .. && python3 tests/tools/ffmpeg.py tests/conformance/failed || true
+
+  windows:
+    name: Windows.MSVC
+    runs-on: windows-latest
+    env:
+      FFMPEG_PATH: msvc\bin\x64\ffmpeg.exe
+
+    steps:
+    - name: Set up MSVC
+      uses: ilammy/msvc-dev-cmd@v1
+
+    - name: Get FFVS-Project-Generator
+      run: Invoke-WebRequest -Uri https://github.com/ShiftMediaProject/FFVS-Project-Generator/releases/download/1.11.6/FFVS-Project-Generator_1.11.6_x64.zip -OutFile FFVS-Project-Generator.zip; Expand-Archive FFVS-Project-Generator.zip
+
+    - name: Get NASM
+      run: Invoke-WebRequest -Uri https://github.com/ShiftMediaProject/VSNASM/releases/download/0.9/VSNASM.zip -OutFile VSNASM.zip; Expand-Archive VSNASM.zip; Start-Process .\VSNASM\install_script.bat
+
+    - name: Get repository
+      uses: actions/checkout@v3
+      with:
+        path: src/main
+
+    - name: Generate visual studio project
+      run: .\FFVS-Project-Generator\project_generate.exe --rootdir=src/main --disable-everything --enable-ffmpeg  --enable-decoder=vvc --enable-parser=vvc --enable-demuxer=vvc --enable-protocol=file --enable-protocol=pipe --enable-encoder=rawvideo --enable-muxer=rawvideo --enable-muxer=md5
```

Comment:
Can do, although it is worth noting as I say above FFVS-Project-Generator actually interprets some arguments differently to the unix ./configure script. Specifically:
* It does not support commas, for example `--enable-muxer=rawvideo,md5` does not work.
* `--disable-everything` and `--disable-all` are synonymous, whereas in the unix version `--disable-everything` disables all components, but not all programs and other aspects of the build.
* Some command line options such as `--disable-asm` are not supported.

I actually kept the two separate because of this, but would you still like to try to have some flags shared? Maybe there could be a workflow-wide variable for shared flags and then job-specific variables. I will also put a comment warning anyone editing the workflow about this problem.

Reply:
Thank you for explaining.  Considering the difference, we can keep these two command lines.
Seems some cases failed. it may be related to memory issues.
Maybe we can think about reducing worker number on windows. https://github.com/ffvvc/tests/blob/87ae9fed1283b1d101ef0c44609cb3739ed81779/tools/ffmpeg.py#L123

Reply:
Why do you think [WPP_B_Sharp_2](https://github.com/ffvvc/FFmpeg/pull/52/checks#step:9:270) and [LTRP_A_ERICSSON_3](https://github.com/ffvvc/FFmpeg/pull/52/checks#step:9:177) are failing? I see the other four fails result from the runner running out of memory, but these appear to complete successfully and then have a md5 mismatch. LTRP_A_ERICSSON_3 failed on my fork as well, where I didn't have any memory issues.

Reply:
There are two+ issues here.
1. Memory allocation issue. We need to reduce memory in code. But the current quick solution is to reduce worker threads.
2. Random failure. It may be related to https://github.com/ffvvc/FFmpeg/issues/26 and https://github.com/ffvvc/FFmpeg/pull/49#issuecomment-1465235997.

Let us fix the first one and merge the PR. Then we can think about how to fix the second one.

Reply:
> Let us fix the first one and merge the PR. Then we can think about how to fix the second one.

👍 Would you like me to make that change to [ffvvc/tests](https://github.com/ffvvc/tests) or will you?